src/classes/REL_Relationships_TDTM.cls

```diff
@@ -72,40 +87,34 @@ public class REL_Relationships_TDTM extends TDTM_Runnable {
         //Keeps track of relationships that have already been deleted to prevent Entity is Deleted error
         Set<Id> hasBeenDeleted = new Set<Id>();
 
-        if (triggerAction == TDTM_Runnable.Action.afterUpdate) {
-            hasRun = true;
-        }
-
-        if (triggerAction == TDTM_Runnable.Action.afterDelete) {
+        if (triggerAction == TDTM_Runnable.Action.AfterDelete) {
             hasBeenDeleted.addAll((new Map<Id, Relationship__c>(newList)).keySet());
         }
         
         Integer i = 0;          // processing counter            
-        for(Relationship__c r : newList) {
+        for (Relationship__c r : newList) {
             // AFTER INSERT
-            if (triggerAction == TDTM_Runnable.Action.afterInsert) {
+            if (triggerAction == TDTM_Runnable.Action.AfterInsert) {
                 if (r.ReciprocalRelationship__c == null) {
                     relationshipsToCreate.add(r);
-                }
-                if (r.ReciprocalRelationship__c != null) {
+                } else {
                     reciprocalsToUpdate.add(r);
                 }
             }
             
             // AFTER UPDATE
-            if (triggerAction == TDTM_Runnable.Action.afterUpdate) {   
+            if (triggerAction == TDTM_Runnable.Action.AfterUpdate) {   
                 if (r.ReciprocalRelationship__c == oldList[i].ReciprocalRelationship__c 
-                && r.RelatedContact__c != null) {
+                        && r.RelatedContact__c != null) {
                     relationshipsToUpdate.add(r);
                     origRelationships.add(oldList[i]);
-                }
-                if (r.RelatedContact__c == null) {
+                } if (r.RelatedContact__c == null) {
```

Comment:
Is there supposed to be a carriage return here (separate if?) or is there supposed to be an "else if" here?

Reply:
I don't want to mess with the existing logic too much here, so I've just added the carriage returns back.